tests/test.py

```diff
@@ -162,27 +162,33 @@ def test_big_result_stored_remotely(self, uuid_mock):
         """
 
         event_with_ingest = {
-            'ingest_meta': {
-                'message_bucket': self.bucket_name
-            },
             'cumulus_meta': {
-                'workflow': 'testing'
+                'workflow': 'testing',
+                "buckets": {
+                    "internal": "testing-internal"
```

Comment:
Can this just refer to the variable defined above, e.g.:
```javascript
"internal": bucket_name,
```
In case we change the name we don't have to change it again in multiple places

Reply:
updated